tests/Application/templates/bundles/SyliusAdminBundle/_scripts.html.twig

```diff
@@ -0,0 +1,2 @@
+{{ encore_entry_script_tags('admin-entry', null, 'admin') }}
+{{ encore_entry_script_tags('sylius-orderCreation-admin', null, 'orderCreation_admin') }}
```

Comment:
It should be done with using template events:
- sylius.admin.layout.stylesheets
- sylius.admin.layout.javascripts
- sylius.shop.layout.stylesheets
- sylius.shop.layout.javascripts

In such case, we don't require additional steps while installing the plugin. It applies to all 4 modified files.